app/assets/javascripts/bulma.js

```diff
@@ -0,0 +1,23 @@
+document.addEventListener('DOMContentLoaded', () => {
+  setupNotificationCloseButton();
+  setupNavbarBurger();
+});
+
+function setupNotificationCloseButton() {
+  document.querySelectorAll('.notification .delete').forEach((closeButton) => {
+    const notification = closeButton.parentNode;
+
+    closeButton.addEventListener('click', () => notification.parentNode.removeChild(notification));
+  });
+}
+
+function setupNavbarBurger() {
+  const navbarBurger = document.querySelector('.navbar-burger');
+
+  navbarBurger.addEventListener('click', (e) => {
+    const menu = document.getElementById(e.target.dataset.target);
+
+    e.target.classList.toggle('is-active');
+    menu.classList.toggle('is-active');
```

Comment:
Inspired by the [offical Bulma example](https://bulma.io/documentation/components/navbar/#navbarJsExample).